doc/source/structures/collections/list.rst

```diff
@@ -50,33 +53,18 @@ Structure
         * - :meth:`REMOVE(index)`
           - None
           - remove item by index
-        * - :attr:`CLEAR`
+        * - :meth:`CLEAR()`
```

Comment:
Technically this clear suffix is part of `Collection` now.  But I don't know if we want to have that much extra clicking to get to the `Enumerable` suffixes (see comment above).

(the same issue arises with the other collections - queue, etc).